.github/workflows/building.yml

```diff
@@ -15,11 +15,14 @@ jobs:
         # python-version: ['3.7']
         # torch-version: [1.11.0]
         # cuda-version: ['cu113']
-        os: [ubuntu-20.04, windows-2019]
+        os: [ubuntu-20.04]
         # support version based on: https://download.pytorch.org/whl/torch/
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
-        torch-version: [1.11.0, 1.12.0, 1.13.0, 2.0.0]
-        cuda-version: ['cu113', 'cu115', 'cu116', 'cu117', 'cu118']
+        #python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        #torch-version: [1.11.0, 1.12.0, 1.13.0, 2.0.0]
+        #cuda-version: ['cu113', 'cu115', 'cu116', 'cu117', 'cu118']
+        python-version: ['3.10',]
+        torch-version: [2.0.0, ]
+        cuda-version: ['cu118',]
```

Comment:
Sorry I did not mean to keep these changes and will revert them. I agree we should ideally test all combination, but we probably do not need to publish all of them in GitHub release.  it is probably ok to drop python 3.7 and 3.8  in the published wheels as these a fairly only version. Python 3.7 that does doe get security fixes (end of life phase) since June 2023, and python 3.8 will stop receiving security fix after October this year. I would also think that very few people still use pytorch 1.11.0 and either use 1.13.0 or 2.x

Reply:
for torch audio , each torchaudio version is compatible with only one torch version (see compatibility matrix at the bottom of the page here)
https://pytorch.org/audio/2.1/installation.html
so that seems to indicate that different torch 2.x version do not have compatible ABI

Reply:
Yeah I also think we could drop 1.13. 

How about lets go with three latest python version, three latest torch versions and three latest CUDA version. Then we could build against:

- python 3.10, 3.11, 3.12
- torch 2.2, 2.3, 2.4
- cuda 118 121 124

I'm not a fan of torchaudio's strategy. Their strategy is ok only if the library itself is not quite updating that much, and it's just about to bump the dependency version.